worker/download.go

```diff
@@ -640,6 +643,9 @@ func (s *slabDownload) nextRequest(ctx context.Context, resps *sectorResponses,
 				s.unusedHostSectors[host] = sectors
 			}
 		}
+		// TODO: track which sectors are being downloaded and prefer
```

Comment:
Reminder to resolve this into issue before merging

Reply:
I implemented a possible solution here https://github.com/SiaFoundation/renterd/pull/1597. I like it because it's simple and it keeps a refactor to a minimum. it's not the best time to make changes to the download code but we should definitely get this fallback mechanism implemented and merged.